d2l-scroll-wrapper.js

```diff
@@ -139,8 +139,8 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-scroll-wrapper">
 			}
 		</style>
 		<d2l-sticky-element class="sticky" disabled="[[_stickyIsDisabled]]">
-			<d2l-table-circle-button class="left action" icon="tier1:chevron-left" on-tap="handleTapLeft" tabindex="-1" aria-hidden="true" type="button"></d2l-table-circle-button>
-			<d2l-table-circle-button class="right action" icon="tier1:chevron-right" on-tap="handleTapRight" tabindex="-1" aria-hidden="true" type="button"></d2l-table-circle-button>
```

Comment:
These overflow buttons weren't keyboard accessible, which meant the focus styles never go applied, so I removed the negative tabindex.